tests/kernels/test_awq.py

```diff
@@ -0,0 +1,37 @@
+import torch
+
+from tests.kernels.utils import opcheck
+from vllm import _custom_ops as ops  # noqa: F401
+from vllm import envs
+
+
+def test_awq_dequantize_opcheck():
+    envs.VLLM_USE_TRITON_AWQ = False
```

Comment:
I don't think you can write env var in this way. use `os.environ["VLLM_USE_TRITON_AWQ"] = value` instread. see https://github.com/vllm-project/vllm/blob/2467b642dd9bde32a334fe5967efd78a53aa49da/vllm/envs.py#L426